CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Minor: Use relevant skill icon for level up notification embeds. (#119)
```

Comment:
Do we want to smaller cosmetic changes like this too?

```suggestion
- Minor: Use relevant skill icon for level up notification embeds. (#119)
- Minor: Change Discord Webhook Slayer notification icon from the Slayer skill icon to the Slayer Helmet icon. (#119)
```

Reply:
Can consider consolidating/generalizing since it's a small cosmetic change

```suggestion
- Minor: Update embed icons for level up and slayer task notifications. (#119)
```

Reply:
I'm happy with your suggestion @iProdigy - thoughts @Felanbird ?